README.md

```diff
@@ -188,10 +188,10 @@ There is also a [sample](https://github.com/fnando/rn-stellar-sdk-sample) that y
 ## Usage
 
 For information on how to use js-stellar-sdk, take a look at the
-[Developers site](https://www.stellar.org/developers/js-stellar-sdk/reference/).
+[Developers site](https://github.com/stellar/js-stellar-sdk/blob/master/docs/reference/readme.md).
 
 There is also API Documentation
-[here](https://www.stellar.org/developers/horizon/reference/index.html).
+[here](https://developers.stellar.org/api/introduction/).
```

Comment:
I think we should also link to the JS documentation for this SDK which lives at: https://stellar.github.io/js-stellar-sdk/. That's often the documentation I want first, but we don't make it very easy to find. 

I think we could be clearer on line 193 that by "API Documentation" we're referring to Horizon's API, and not the SDK's API, which is not at that link.

I realize both of these things are out-of-scope of this PR, but somewhat relevant.